src/Integration/Telemetry/TelemetryHelper.cs

```diff
@@ -42,6 +42,8 @@ public static TelemetryPayload CreatePayload(TelemetryData telemetryData, DateTi
                 NumberOfDaysSinceInstallation = now.DaysPassedSince(telemetryData.InstallationDate),
                 NumberOfDaysOfUse = telemetryData.NumberOfDaysOfUse,
                 IsUsingConnectedMode = isConnected,
+                SystemDate = now,
+                InstallDate = telemetryData.InstallationDate,
```

Comment:
We have too many layers (3?) of conversion and apparently it is easy for me to forget one of them :)

Reply:
Can we simplify to have less layers?

Reply:
Yes, but I will do it some other time...